### PR TITLE
bump docs

### PR DESCRIPTION
**Pull request by oxarbitrage**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,8 +28,7 @@ WORKDIR /bitshares-core
 
 # Compile
 RUN \
-    git submodule sync --recursive && \
-    git submodule update --init --recursive && \
+    git submodule foreach git pull && \
```
Comment (abitmore) [inline Dockerfile]: Add `--recursive`? IIRC Docker doesn't build from scratch everytime.

**Comment by oxarbitrage** [comment]
tried all i can but it keeps failing:

https://cloud.docker.com/swarm/bitshares/repository/registry-1.docker.io/bitshares/bitshares-core/builds/e66bc4ac-8029-4595-829e-f58a79509010

```
 ---> Running in 7efec6884820
Entering 'docs'
fatal: Not a git repository: /src/b7wffqqdqxbtmdtata3ymra/.git/modules/docs
Stopping at 'docs'; script returned non-zero status.
Removing intermediate container 7efec6884820
```

**Comment by oxarbitrage** [comment]
we can try to merge bump the docs only and then try changes at Dockerfile with the last docs version merged.

**Comment by oxarbitrage** [comment]
```
 ---> Running in bf5d7b414691
fatal: Not a git repository: /src/bzflbuh22i6ugy6eitucirb/.git/modules/docs
Unable to find current revision in submodule path 'docs'
---> 0d42a44a7be9
Removing intermediate container bf5d7b414691
Step 8/19 : RUN git submodule sync --recursive && git submodule update --init --recursive && cmake -DCMAKE_BUILD_TYPE=Release . && make witness_node cli_wallet && make install && mkdir /etc/bitshares && git rev-parse --short HEAD > /etc/bitshares/version && cd / && rm -rf /bitshares-core
---> Running in d70c933aa10a
Synchronizing submodule url for 'docs'
fatal: Not a git repository: /src/bzflbuh22i6ugy6eitucirb/.git/modules/docs
fatal: Not a git repository: /src/bzflbuh22i6ugy6eitucirb/.git/modules/docs
fatal: Not a git repository: /src/bzflbuh22i6ugy6eitucirb/.git/modules/docs
fatal: Not a git repository: /src/bzflbuh22i6ugy6eitucirb/.git/modules/docs
Cannot chdir to , the toplevel of the working tree
Synchronizing submodule url for 'libraries/fc'
fatal: Not a git repository: /src/bzflbuh22i6ugy6eitucirb/.git/modules/libraries/fc
fatal: Not a git repository: /src/bzflbuh22i6ugy6eitucirb/.git/modules/libraries/fc
fatal: Not a git repository: /src/bzflbuh22i6ugy6eitucirb/.git/modules/libraries/fc
fatal: Not a git repository: /src/bzflbuh22i6ugy6eitucirb/.git/modules/libraries/fc
```

**Comment by oxarbitrage** [comment]
work on this continued by @pmconrad at https://github.com/bitshares/bitshares-core/pull/1025 and https://github.com/bitshares/bitshares-core/pull/1023. issue also https://github.com/bitshares/bitshares-core/issues/1022

**Pull request closed without merging**